package.json

```diff
@@ -1,32 +1,32 @@
 {
     "name": "universal",
     "description": "Cross platform, core components of the GPII personalization infrastructure.",
-    "version": "0.1.0",
+    "version": "1.0.0",
```

Comment:
just asking ... are we releasing our 1.0.0 version of the GPII with this pull request?